<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>天天协议代码生成器</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListView" name="xml_list_view">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>331</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="generate_one_btn">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>410</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>生成选中协议</string>
    </property>
   </widget>
   <widget class="QPushButton" name="generate_all_btn">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>410</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>生成所有协议</string>
    </property>
   </widget>
   <widget class="QPushButton" name="select_xml_path_btn">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>20</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>选择协议文档目录</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="select_xml_path_txt">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="select_cpp_path_txt">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="select_cpp_path_btn">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>60</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>选择cpp生成目录</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="select_lua_path_txt">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="select_lua_path_btn">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>100</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>选择lua生成目录</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="generate_log_txt">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>160</y>
      <width>256</width>
      <height>231</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;生成日志&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
